components/common/AppDrawer.tsx

```diff
@@ -1,13 +1,16 @@
-import ClubRegistryLogo from "@/public/images/drawer/club-registry.png";
-import KularbFahLogo from "@/public/images/drawer/kularb-fah.png";
+import ClubRegistryLogo from "@/public/images/drawer/club-registry.svg";
```

Comment:
Using SVGs instead of PNGs would result in infinitely higher image quality from a vector-based image. Doing this would result in developers not having to worry if their app image is too low for future times like when technology has advanced to 128k monitors.